cmake/IBAMRConfig.cmake.in

```diff
@@ -19,20 +19,20 @@
 SET(MPI_ROOT "@MPI_ROOT@")
 FIND_PACKAGE(MPI REQUIRED)
 
-SET(Boost_FOUND "@Boost_FOUND@")
-SET(BOOST_ROOT "@BOOST_ROOT@")
-IF("${Boost_FOUND}")
+IF(NOT @IBAMR_USE_BUNDLED_Boost@)
+  SET(BOOST_ROOT "@BOOST_ROOT@")
```

Comment:
I think `Boost_ROOT` is used as the root.
```suggestion
  SET(BOOST_ROOT "@Boost_ROOT@")
```

Reply:
Yes, this should always be `Boost` - fixed.